index.js

```diff
@@ -0,0 +1,3 @@
+function getRandomNumber() {
```

Comment:
Lets improve `getRandomNumber` function, please look at this [page](https://stackoverflow.com/questions/1527803/generating-random-whole-numbers-in-javascript-in-a-specific-range#:~:text=240-,Math.random(),-Returns%20an%20integer)